mod-copycat/src/main/resources/karate-config.js

```diff
@@ -60,6 +62,16 @@ function fn() {
       name: 'testing_admin',
       password: 'admin'
     }
+  } else if(env == 'eureka'){
+    config.baseUrl = 'https://folio-edev-dojo-kong.ci.folio.org:443';
+    config.baseKeycloakUrl = 'https://folio-edev-dojo-keycloak.ci.folio.org:443';
+    config.admin = {
+      tenant: 'diku',
+      name: 'diku_admin',
+      password: 'admin'
+    }
+    config.clientSecret = karate.properties['clientSecret'];
+    config.login = karate.read('classpath:common/eureka/login.feature')
```

Comment:
we probably shouldn't overwrite the existing login.feature because I've been told that /authn/login is a deprecated endpoint and /authn/login-with-retry is up-to-date. This won't affect us anyway because the functionality is comparable